testsuite/features/secondary/min_bootstrap_negative.feature

```diff
@@ -61,9 +70,12 @@ Feature: Negative tests for bootstrapping normal minions
      And I enter "root" as "user"
      And I enter "linux" as "password"
      And I click on "Bootstrap"
-     And I wait until I see "ssh: connect to host" text
-     Then I should not see a "GenericSaltError" text
-     And I should see a "port 11: Connection refused" text or "port 11: Invalid argument" text
+     And I wait until I see "An error has occurred during salt execution: unable to establish ssh connection." text
+     Then I should see a "Details" link
+     When I follow "Details" in the content area
+     Then I should see "Error Details" text
+     And I should see "Standard Error" text
+     And I should see "ssh: connect to host uyuni-minion-ubuntu port 11: Connection refused" in the textarea
```

Comment:
I would say the name is wrong : `uyuni-minion-ubuntu` should probably be have the hostname of `sle_minion`. I suggest you just check for the text "Connection refused" and that should be enough.